src/assets/CollectiblesController.test.ts

```diff
@@ -27,7 +27,7 @@ const MAINNET_PROVIDER = new HttpProvider(
 const OWNER_ADDRESS = '0x5a3CA5cD63807Ce5e4d7841AB32Ce6B6d9BbBa2D';
 const SECOND_OWNER_ADDRESS = '0x500017171kasdfbou081';
 
-const OPEN_SEA_HOST = 'https://api.opensea.io';
+const OPEN_SEA_HOST_PROXY = 'https://proxy.metaswap.codefi.network/opensea/v1';
```

Comment:
Could you add tests for the fallback behaviour as well?

Reply:
Waiting to go back and complete tests til we have alignment on the rest.

Reply:
added a test for the fallback behavior here: https://github.com/MetaMask/controllers/pull/805/commits/794069034080dfd7762402412f1d059204b1961a